aws_jobs/cyhy-data-extract.py

```diff
@@ -22,7 +22,12 @@
 
 import re
 import sys
-from ConfigParser import SafeConfigParser
+
```

Comment:
I'd probably move this after `import os` since `SafeConfigParser` is the thing being imported.

Reply:
Adjusted that and any similar imports per your feedback.